microsoft-edge/devtools-guide-chromium/sample-code/sample-code.md

```diff
@@ -5,49 +5,47 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 02/01/2022
+ms.date: 02/03/2022
 ---
 # Sample code for DevTools
 
-Most of the sample code that's used by the DevTools documentation is stored in the following GitHub repos.
+The sample code that's used by the DevTools documentation is mainly in the [MicrosoftEdge/Demos](https://github.com/MicrosoftEdge/Demos) repo at GitHub.
+
+<!-- A few demos are at other locations, such as Glitch. -->
 
 
 <!-- ====================================================================== -->
-## GitHub repos for samples
+## Demo webpage with accessibility issues
 
-The [MicrosoftEdge/Demos](https://github.com/MicrosoftEdge/Demos) repo contains most of the demo pages or sample code that's used by the documentation.
+This demo webpage is useful for exploring various DevTools features, such as the **Elements** tool.
 
-The [MicrosoftEdge/DevToolsSamples](https://github.com/MicrosoftEdge/DevToolsSamples) repo contains several demos that are used by the documentation, including for **3D View**, **Inspect tool**, **Console**, and accessibility testing.
+1. Open the [Demo webpage with accessibility issues](https://MicrosoftEdge.github.io/Demos/devtools-a11y-testing/) in a new window or tab.
 
-The demo pages below are stored in those repos, and are useful for exploring tools such as **Elements** and **Sources**.
+1. Right-click anywhere in the rendered webpage and then select **Inspect**.  DevTools opens next to the demo webpage.
 
+   <!-- Or, press `F12`, `Ctrl`+`Shift`+`I` (on Windows/Linux), or `Command`+`Option`+`I` (on macOS). -->
```

Comment:
In Inspect always available?  Can everyone reach it ok?  Do we need to show keyboard shortcuts to open DevTools?  @captainbrosset

Reply:
Yes Right-click > Inspect is always available in Edge. I believe we have many other places in the doc that already talk about the keyboard shortcuts, so maybe we don't need to mention them here. Either way works for me.